packages/react-core/src/components/Form/FormSection.tsx

```diff
@@ -2,19 +2,23 @@ import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/Form/form';
 import { css } from '@patternfly/react-styles';
 
-export interface FormSectionProps extends React.HTMLProps<HTMLDivElement> {
+export interface FormSectionProps extends Omit<React.HTMLProps<HTMLDivElement>, 'title'> {
   /** Content rendered inside the section */
   children?: React.ReactNode;
   /** Additional classes added to the section */
   className?: string;
+  /** Title for the section */
+  title?: React.ReactNode;
 }
 
 export const FormSection: React.FunctionComponent<FormSectionProps> = ({
   className = '',
   children,
+  title = '',
   ...props
 }: FormSectionProps) => (
   <section {...props} className={css(styles.formSection, className)}>
+    {title && <div className={css(styles.formSectionTitle, className)}>{title}</div>}
```

Comment:
Should we allow `titleComponent` here (and update the core docs) to allow a heading or other element to be used?

Reply:
That'll be an easy enough change for a follow up after the update to core is made, but I can also open a core issue and pause on this until its done. either way

Reply:
PR is here - https://github.com/patternfly/patternfly/pull/3973

Reply:
@nicolethoen the docs were updated in core